### PR TITLE
add Awesome-IIIF

### PR DESCRIPTION
**Pull request by julsraemy**
The Awesome IIIF GitHub repository seems like a good pointer to add to the DH Awesome list rather than duplicating all IIIF-compliant resources in my opinion.

- [x] I have read and understood the [contribution guidelines](https://github.com/writing-resources/awesome-scientific-writing/blob/main/CONTRIBUTING.md).
- [ ] Table of contents has been updated (if a section is added / removed).
- [x] Contents have been sorted alphabetically.

### CODE REVIEW

**Review by diegosiqueir4 — APPROVED** [review]
This is indeed a very good idea, @julsraemy. Thanks for the contribution!

**Review by maehr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ This is a curated list of tools, resources, and services supporting the Digital
 ## Other Resources
 
 - [Awesome Digital History](https://github.com/maehr/awesome-digital-history) - A curated list of awesome things related to digital history.
+- [Awesome IIIF](https://github.com/IIIF/awesome-iiif) - A curated list of awesome resources related to the International Image Interoperability Framework (IIIF)
```
Comment (maehr) [inline README.md:115]: @julsraemy Add a dot at the end of the line to make the linter happy. Thanks for the contribution!

**Review by maehr — CHANGES_REQUESTED** [review]

**Review by julsraemy — COMMENTED** [review]
Add `.` and make @maehr and Linter happy

File: README.md (hunk 1 above)
Comment (julsraemy) [reply README.md:115]: Ok!

**Review by maehr — APPROVED** [review]

**Pull request merged**